app/components/rating/rating.twig

```diff
@@ -0,0 +1,7 @@
+{% macro default() %}
```

Comment:
If you want, you could simply enhance this component with a numeric parameter `default(rank)` an loop through the value with http://twig.sensiolabs.org/doc/2.x/tags/for.html